module/VuFind/src/VuFind/ILS/Driver/OAuth2TokenTrait.php

```diff
@@ -100,7 +100,7 @@ public function getNewOAuth2Token(
                 'Problem getting authorization token: Bad status code returned'
             );
         }
-        $tokenData = json_decode($response->getContent(), true);
+        $tokenData = json_decode($response->getBody(), true);
```

Comment:
Just curious what motivated this change -- it does seem to make more sense to use the decoded content rather than the raw content, but just wanted to be sure it wasn't intentionally working the other way for some context. Also note that getContent is still being used both earlier and later in error/log messages in this same code. Should this be made more uniform?

Reply:
json_decode could fail on raw http response content especially when Content-encoding = deflate. So I changed it, because I really experienced this issue... And yes, it should be consistant, also getBody() is always returning string type unlike getContent. It is done now.

Reply:
Makes sense -- thanks!